charts/activemq/templates/_helpers.tpl

```diff
@@ -48,6 +48,7 @@ Selector labels
 {{- define "activemq.selectorLabels" -}}
 app.kubernetes.io/name: {{ include "activemq.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
+app.kubernetes.io/component: {{ template "activemq.name" . }}
```

Comment:
technically it will not change the fact that component will have the proper component label set, but conceptually will use it as a selector in the service and that's not the purpose of this additional label

it's better if you add it in the `.labels` named template